src/nvim/testdir/test_makeencoding.vim

```diff
@@ -13,12 +13,19 @@ endif
 
 let s:script = 'test_makeencoding.py'
 
-let s:message_tbl = {
+if has('iconv')
+  let s:message_tbl = {
```

Comment:
when is iconv not available? The MSVC builds have it now.

Reply:
https://github.com/neovim/neovim/pull/8160#issuecomment-377104244
The MSVC logs show that the iconv is not found, `-iconv` in Features.
In functional tests, I get this

```
  [----------] Running tests from C:/projects/neovim/test/functional\legacy\083_tag_search_with_file_encoding_spec.lua
  [ RUN      ] tag search with !_TAG_FILE_ENCODING skipped (missing iconv): 0.00 ms SKIP
  [----------] 1 test from C:/projects/neovim/test/functional\legacy\083_tag_search_with_file_encoding_spec.lua (47.00 ms total)
```

Reply:
Was this rebased recently?

Reply:
Rebasing did nothing. The MSVC build log still report that iconv.h is not found.

Reply:
My mistake, I was thinking of libintl, not iconv.